cmake/Modules/FindMfx.cmake

```diff
@@ -0,0 +1,53 @@
+# * Try to find libvpl/libmfx, prefers vpl
```

Comment:
If it's two different library. Whatever their similarity, it should two different finders.

Reply:
These are two different dispatchers for the same API, as such either provides the functionality we need. This was an mainly an attempt to make it work on ubuntu 20, which doesnt provide the vpl dispatcher. However since ubuntu 20 doesnt provide av1 definitions it doesnt end up working. So we could remove the mfx fallback as ubuntu 22 provides vpl.